.storybook/main.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
   framework: '@storybook/react',
   staticDirs: ['./public'],
   webpackFinal: async (config) => {
+    //use commonjs entry point for "@reach" packages
+    config.resolve.alias['@reach/auto-id'] = require.resolve('@reach/auto-id');
```

Comment:
out of curiosity why is this necessary?

Reply:
our storybook uses commonjs and doesn't seem to handle mjs files. 
<img width="730" alt="Screen Shot 2022-10-17 at 12 59 15 PM" src="https://user-images.githubusercontent.com/36055303/196238389-1afdf4fd-3e03-489f-bd25-b3b62f115110.png">

`@reach/auto-id` does export a commonjs version. But its package.json is set to `"module": "./dist/reach-auto-id.mjs",` and `"main": "./dist/reach-auto-id.cjs.js",`. I believe Webpack, used by Storybook, by default will try to use "[module" over "main"](https://github.com/webpack/webpack/issues/5756). Alternatively, we could also do `config.resolve.mainFields = ['main', 'module'];` but I thought the alias would be a bit clearer.